pkg/ccl/importccl/import_table_creation.go

```diff
@@ -146,6 +146,7 @@ func MakeSimpleTableDescriptor(
 	tableID descpb.ID,
 	fks fkHandler,
 	walltime int64,
+	sessionData *sessiondata.SessionData,
```

Comment:
hmm I'm a little confused why we're plumbing the session data through this method. Does `DefaultIntSize` get used somewhere that stems from this method? My understanding is that by the time we reach here the `*tree.CreateTable` column definitions already have the types set to the correct int types.

We explicitly set the types for the bundle formats, do we need to do anything for `IMPORT TABLE`?

Reply:
You're correct! We don't need to plumb session data here.

Table creation for `IMPORT TABLE` happens in the planning stage, where we have access to the session data, hence no changes are needed.

Reply:
👍